.github/workflows/testcafe-testing-library.yml

```
@@ -13,13 +13,13 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [14, 12, 10]
+        node: [10, 12, 14, 16]
```

Node 15 is still supported until June 1
```suggestion
        node: [10, 12, 14, 16]
```

---

We shouldn't support odd Node versions, only when they're `latest`

---

As far as I understand from [Node's LTS schedule](https://github.com/nodejs/Release), 15 is actually still supported right now, even though 16 (which will eventually be upgraded to an LTS) has been released since

---

Odd Node versions should never be officially supported

> Production applications should only use Active LTS or Maintenance LTS releases.

https://nodejs.org/en/about/releases

---

Node 15 is EOL too now